src/ui/server/routes/wellknown/index.test.ts

```diff
@@ -0,0 +1,19 @@
+import { get } from '../../test-mocks/mock-router';
+import { ROUTES } from '../../constants';
+import wellknownGet from '../../controllers/wellknown/security';
+
+describe('routes/.well-known', () => {
```

Comment:
```suggestion
describe('routes/well-known', () => {
```

Can we rename the directory to be `well-known` as well please

Reply:
Can we rename the directory as well please @abhi-markan

Reply:
Directory is renamed to `well-known`.